packages/governance/docs/AttackGuide

```diff
@@ -0,0 +1,101 @@
+# An Attacker's guide to Agoric Governance
```

Comment:
It looks like you intend this to be a Markdown document. Please rename the file to end in `.md` and github will offer to render it.

Reply:
done